homeassistant/components/roku/media_player.py

```diff
@@ -67,7 +67,7 @@ def unique_id(self) -> str:
     def state(self) -> str:
         """Return the state of the device."""
         if self.coordinator.data.state.standby:
-            return STATE_STANDBY
+            return STATE_OFF
```

Comment:
This is a breaking change.

Reply:
yah i fixed the checkboxes as i did end up realizing and added paragraph as well